project/tests/testamity.py

```diff
@@ -0,0 +1,134 @@
+import unittest
+from app.Amity import Amity, Rooms, Office, LivingSpace
+from app.people import Person, Fellow, Staff
+
+
+class OfficeSpaceAllocationTests(unittest.TestCase):
+
+    def setUp(self):
+        self.facility = Amity()
+        self.room = Rooms()
+        self.room2 = Rooms()
+
+    def test_create_single_office(self):
+        '''Tests creating a single office room'''
+        self.assertEqual(0, len(self.facility.rooms_list))
+        self.facility.create_room(
+            {"room_type": "office", "room_name": "Krypton"})
+        num = len(self.facility.rooms_list)
+        self.assertEqual(1, num)
+
+    def test_create_single_living_room(self):
+        ''''Tests creating a single living space'''
+        self.assertEqual(0, len(self.facility.rooms_list))
+        self.facility.create_room(
+            {"room_type": "living", "room_name": "PHP"})
+        self.assertEqual(1, len(self.facility.rooms_list))
+
+    def test_create_multiple_offices(self):
+        self.assertEqual(0, len(self.facility.rooms_list))
+        self.facility.create_room(
+            {"room_type": "office",
+                "room_name": ["Hogwarts", "Bumblebee", "Cyan"]})
+        self.assertEqual(3, len(self.facility.rooms_list))
+
+    def test_create_multiple_living_spaces(self):
+        self.assertEqual(0, len(self.facility.rooms_list))
+        self.facility.create_room(
+            {"room_type": "living",
+                "room_name": ["Java", "Python", "Android"]})
+        self.assertEqual(3, len(self.facility.rooms_list))
+
+    def test_create_person(self):
+        '''Asserts that person is added to a list and they are in the list'''
+
+        s_fname = "Percila"
+        f_fname = "Paul"
+        self.assertEqual(0,
+                         len(self.facility.persons_list))
+
+        self.facility.create_person(s_fname, "Njira", "Staff", "N")
+        self.assertEqual(1,
+                         len(self.facility.persons_list))
+
+    def test_person_values_added_are_of_correct_types(self):
+
+        self.assertEqual(0,
+                         len(self.facility.persons_list))
+        self.assertRaises(ValueError,
+                          self.facility.create_person, 456, "Muthama", "Fellow", "Y")
+        self.assertRaises(ValueError,
+                          self.facility.create_person, "Paul", 89, "Fellow", "Y")
+        self.assertRaises(TypeError,
+                          self.facility.create_person, "Percila", "Njira", "Fellow", 0)
+
+    def test_add_person_to_room(self):
+        r_name = "PHP"
+        f_fname = "Paul"
+        self.assertEqual(0, len(self.facility.allocated_persons))
+        self.facility.create_room(
+            {"room_type": "living", "room_name": r_name})
+        self.facility.create_room(
+            {"room_type": "office", "room_name": 'krypton'})
```

Comment:
Your are supposed to mock what you have implemented at code block 69-72. checkout this mock documentation https://docs.python.org/2/library/unittest.html